Add an aside for symlink errors on Windows (#20)
pesde-pkg/pesde: docs/src/content/docs/installation.mdx

      pesde should now be installed on your system. You may need to restart your
      computer for the changes to take effect.

      [dev-mode-guide]: https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development

>       <Aside type="caution" title="Warning">

Reviewer: What's wrong with the default `Caution`?

      pesde uses symlinks which are an administrator-level operation on Windows.

>       To ensure proper functionality, enable [Developer Mode][dev-mode-guide].<br/><br/>

Reviewer: ```suggestion
      To ensure proper functionality, enable [Developer Mode](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development).
      
      
```

Let's inline this URL and use newlines instead of `<br>`s.

>       If you are getting errors such as `'Failed to symlink file, a required privilege is not held by the client'`, then enabling this setting will fix them.

Reviewer: ```suggestion
      If you are getting errors such as `Failed to symlink file, a required
      privilege is not held by the client`, then enabling this setting will fix
      them.
```

Let's break this sentence up a bit, and remove the unnecessary quotes.

>       If you are getting errors such as <strong>'Failed to symlink file, a required privilege is not held by the client'</strong>, then enabling this setting will fix them.

Reviewer: ```suggestion
      If you are getting errors such as `Failed to symlink file, a required privilege is not held by the client`, then enabling this setting will fix them.
```

We should probably use an inline code block for this.

      pesde should now be installed on your system. You may need to restart your
      computer for the changes to take effect.

>       [dev-mode-guide]: https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development
> 
>       <Aside type="caution">

Reviewer: ```suggestion
      <Aside type="caution">
```
This still needs removal.

</Steps>

> <Aside type="caution" title="Warning">

Reviewer: Same as before, why change the title?
